git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,810 @@
+
+As you work on your project, files will cycle through these different states. When you make a change to a file, that change is in your working directory until you restart the `add` and `commit` cycle.
+
+
+**Is there a better way to think of this?** Lots of people have favorite metaphors for how Git works. Before checking out the selection on the next page, do you have any ideas for possible ways to understand the `add` and `commit` process in Git?
+
+### A few metaphors for Git
+Metaphors for Git abound, and no metaphor has been great enough to become the "standard" example. If you are looking for a good enough metaphor to help you understand what Git is, here are some to consider:
+
+* A [poetry metaphor](https://www.youtube.com/watch?v=BCQHnlnPusY) video on YouTube ~14 minutes
+
+* A [musical metaphor](https://www.youtube.com/watch?v=S9Do2p4PwtE) video on YouTube ~5 minutes
+
+* A [metaphor about building a house](https://towardsdatascience.com/a-simple-story-to-explain-version-control-to-anyone-5ab4197cebbc) blog post
+
+* A [metaphor about photo albums](https://www.freecodecamp.org/news/git-the-laymans-guide-to-understanding-the-core-concepts/) article
+
+Remember a metaphor doesn't need to be perfect to be helpful. If you find or come up with one that helps you understand the workflow, use it!
+
+
+### Quiz: Git workflow
+
+What does the `git commit` command do?
+
+ [[ ]] Stages files so that their current version can be recorded.
+ [[ ]] Creates a record of the current version of all files in the repository.
+ [[X]] Creates a record of the current version of all of the staged files in the repository.
+***
+<div class = "answer">
+The `git commit` command will only create a record of the files that you have *already staged.*
+
+There are options you can use to automatically stage all files. Entering `git commit -a` or `git commit --all` will automatically stage all files that Git knows about and then commit them. But it will not save any new files that Git doesn't yet know about.
+</div>
+***
+
+## Additional Resources
+
+This module was based on examples from three of [Software Carpentry](https://software-carpentry.org)'s [lessons on using Git](https://swcarpentry.github.io/git-novice/). The story of Dracula and Wolfman's planned trip to Mars is borrowed from them, and more examples are available in the original lessons:
+
+ - Lesson 3: [Creating a Repository](https://swcarpentry.github.io/git-novice/03-create/index.html)
+ - Lesson 4: [Tracking Changes](https://swcarpentry.github.io/git-novice/04-changes/index.html)
+ - Lesson 6: [Ignoring Things](https://swcarpentry.github.io/git-novice/06-ignore/index.html)
+
+This module is also the third is a series of modules about Git created by Arcus Education and friends. While we suggest continuing with the next module in the series, [Exploring a Project's History](link/goes/here) the entire collection is available on the [Arcus website](link/goes/here)
```

Comment:
Holding this out as a last change.  Also, Arcus is distinct from the R25 work and most people working with this material won't be working with Arcus.  So, maybe leave off the "entire collection" bit for now?

Reply:
Deleted entire paragraph, though we may later want to add some version of it back in.